src/hypothesis/settings.py

```diff
@@ -257,8 +259,41 @@ def __exit__(self, *args, **kwargs):
         default_context_manager = self.defaults_stack().pop()
         return default_context_manager.__exit__(*args, **kwargs)
 
-    default = DefaultSettings()
+    @staticmethod
+    def register_profile(name, **kwargs):
```

Comment:
So in my head you would use explicit Settings objects here rather than kwargs, but I could be persuaded that kwargs are the way to go, or to support both modes. Is there a reason you wanted to do this as kwargs?